### DISCUSSION TITLE
stats not showing

### DISCUSSION THREAD

**Discussion by barrettluke** [FelixKratz/SketchyBar#348] [Q&A] [answered]
I am new to sketchybar and still trying to figure it out. I have some things like clock, battery, and spaces working but for the life of me I can't get stats to show. I ran the sketchybar --query default-menu-items to make sure of the name and that it is running and it gave me the following:  "Stats,GPU_mini",
	"Stats,Disk_mini",
	"Stats,Sensors_sensors",
	"Stats,Network_speed",
	"Stats,RAM_mini",
	"Stats,CPU_mini"
. I then added to my sketchybarrc file the following line sketchybar --add alias "Stats,RAM_Mini" right  and it isn't showing up ? I tried to run it from the command line and got that it already exists? I am running this all on a M1 mac pro,  16GB ram, macOS Ventura 13.0.

**Comment by FelixKratz** [comment, marked as answer]
The command you provided has `Mini` in uppercase, while it is lowercase in the query, try:

```bash
sketchybar --add alias "Stats,RAM_mini" right 
```
